src/app/users/users.component.html

```diff
@@ -1,11 +1,36 @@
 <div *ngIf="isDataReady" class="divSettings col-lg-8 col-md-10 col-sm-12 col-xs-12">
-  <h2>
-    <span>
-      <b>{{memberName}}</b>
-    </span>
-  </h2>
-  <!--  -->
-  <br>
+  <div>
+    <div>
+      <div style="font-size: 24px">
+        <b>{{info.NAME}}</b>
+      </div>
+      <div style="font-size: 20px">
+        {{info.DISPLAYNAME}}
```

Comment:
in future we may want to make explicit that this is the DisplayName

Reply:
ok, at this moment, only a few people have this parameter active, so there would be an empty field below each name